### ISSUE TITLE
Trying to reproduce results in the paper

### ISSUE THREAD

**Issue by rusonyo666** [mmasana/FACIL#33]
Hello !

In the paper [Class-incremental learning: survey and performance evaluation on image classification](https://ieeexplore.ieee.org/abstract/document/9915459), there are several extensive experiments shown in the figures.
 
While I was trying to use the code and run the experiments, the results are a bit different from the paper's results. 

Is it possible to specify the config setting to run these experiments? such as random seeds, learning rates (starting, min and decreasing factors) or weight decays.

For the paper results, I'm mostly regarding to Fig. 7 and Fig. 8 in the paper. The experiments I ran using the code couldn't reach ~80% accuracy on cifar-100 after the first task (10 class), so I'm wondering if I'm missing something on this.

Thank you in advance !!

**Comment by mmasana** [comment]
Hi @rusonyo666 ,

I see that you closed the issue, so I hope you found the details you were looking!

Best,
Marc
